wweval/R/eval_fit.R

```diff
@@ -203,42 +202,50 @@ eval_fit_hosp <- function(config_index,
     raw_input_hosp_data = raw_input_hosp_data,
     forecast_date = forecast_date,
     table_of_exclusions = table_of_exclusions
-  )
+  ) |>
+    dplyr::mutate(
+      forecast_date = forecast_date
```

Comment:
Same here, adding a new variable only?

Reply:
Similar to above it feels like adding the `forecast_date` column belongs inside `get_input_hosp_data`. But also I think DRY could be improved here. Lot of duplication between `eval_fit_hosp` and `eval_fit_ww`.

Reply:
The reason for the separate functions is that they used to call different models with different parameters and it was just easier to separate them. Now that this is just a switch, I think a second PR could make this more DRY by just writing a single function `eval_fit` that takes an argument for `include_ww` or not. 

I see this as a separate PR though, with this PR focused on getting the basic functionality working. Will add to the checklist.

Reply:
And in this PR will include in `get_input_hosp_data()`